hack/quick_install.sh

```diff
@@ -65,7 +65,6 @@ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/relea
 kubectl wait --for=condition=available --timeout=600s deployment/cert-manager-webhook -n cert-manager
 cd ..
 # Install KFServing
-kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml
-
+for i in 1 2 3 ; do kubectl apply -f install/${KFSERVING_VERSION}/kfserving.yaml && break || sleep 15; done
```

Comment:
Thanks @NikeNano ! Can you help leave a comment here with the issue we described on the PR ?

Reply:
Update @yuzisun also updated to retry 5 times to be > 1 minute